### PR TITLE
Translate documentation into English

### LINKED ISSUE #19: Translate documentation into English

### PR DESCRIPTION
**Pull request by baev**
fixes #19

### CODE REVIEW

**Comment by lanwen** [comment]
mb not to remove ru, but move it to README_ru.md?

**Comment by baev** [comment]
IMO twice harder to support both languages

File: README.md [hunk 1]
```diff
@@ -124,47 +120,41 @@ assertThat(proxyProperties.getHost(), equatlTo("proxy.yandex.ru"));
...
+You can change this order using an example [download configurations depending on environment][create-multi-file-configuration] or create [your own property loader][creation-custom-property-provider].
 
-Уже доступен провайдер, позволяющий превращать пути вида
+Already available  property loader which convert paths like this:
 ```java
 @Resource.Classpath("${system.file.name}.path.${map.scope.value}.properties")
 ```
-в реальные пути. Подключить его можно аннотацией
+
+to real pathes. You can enable it using annotation `@With`:
```
Comment (lanwen) [inline README.md]: paths

**Comment by lanwen** [comment]
say when rdy to merge any

**Comment by baev** [comment]
maybe move docs to wiki?

**Comment by lanwen** [comment]
agree. On readme only quickstart

**Comment by lanwen** [comment]
moved all to wiki: edit there

**Pull request closed without merging**